src/utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module wordcount/utils
+ */
+
+/**
+ * Returns plain text representation of an element and it's children. The blocks are separated by a newline (\n ).
+ *
+ * **Note:** Function walks through the entire model, which might be very spread. There should be considered throttling it during usage.
+ *
+ * @param {module:engine/model/element~Element} element
+ * @returns {String} Plain text representing model's data
+ */
+export function modelElementToPlainText( element ) {
+	if ( element.is( 'text' ) || element.is( 'textProxy' ) ) {
```

Comment:
According to the docs this isn't allowed. The element is an element. Please report a followup ticket to clean this up.

Reply:
My other problem with this is that we already have `viewItemToPlainText` in ckeditor5-clipboard. Ideally, those functions should be kept together, but I don't know where would it be. Engine?

Reply:
I'd go with `Engine`. Ps. this method would be suitable also for other features (AFAIR: `Mention` or `TextWatcher` in particular and maybe the `Autoformat` - the latter should use `TextWatcher` API anyway).

Reply:
However this method might soon be changed to follow requirements from #5.